docs/src/userguide/language_basics.rst

```diff
@@ -404,20 +581,48 @@ returns small results.
 
 If all possible return values are legal and you
 can't reserve one entirely for signalling errors, you can use an alternative
-form of exception value declaration::
+form of exception value declaration
 
-    cdef int spam() except? -1:
-        ...
+.. tabs::
+
+    .. group-tab:: Pure Python
+
+        .. code-block:: python
+
+            @cython.exceptval(-1, check=True)
+            def spam() -> cython.int:
+                ...
+
+    .. group-tab:: Cython
+
+        .. code-block:: cython
+
+            cdef int spam() except? -1:
+                ...
 
 The "?" indicates that the value ``-1`` only signals a possible error. In this
```

Comment:
there should be text about the `check` attribute

Reply:
… and the text would best be placed in the tab to make it change with the syntax display.

Reply:
done.